src/index.html

```diff
@@ -3,10 +3,474 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+
+        <div class="header__logo">
+          <img src="./images/logo.png" alt="logo">
+        </div>
+
+        <nav class="nav">
+          <ul class="nav__menu">
+
+            <li class="nav__list">
+              <a href="#" class="nav__link is-active">apple</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link">samsung</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link">смартфоны и телефоны</a>
+            </li>
+
+            <li class="nav__list">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                ноутбуки и компьютеры
+              </a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link">гаджеты</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link">планшеты</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link">фото</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link">видео</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <div class="catalog main__container">
+
+        <div class="card catalog__card" data-qa="card">
+          <div class="card__photo">
+            <img
+              src="images/imac.jpeg"
+              height="134px"
+              width="160px"
+              alt="imac"
+            >
+          </div>
+
+          <div class="title">
+            <h2 class="title__info">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+            <p class="title__code">
+              Код товара: 195434
+            </p>
+          </div>
+
+          <div class="review">
+            <div class="review__stars">
+              <div class="review__item"></div>
+              <div class="review__item"></div>
+              <div class="review__item"></div>
+              <div class="review__item"></div>
+              <div class="review__item review__item--last_color"></div>
+            </div>
```

Comment:
I think we need here somethink like in HW "stars", with modeficator stars--4, or review__stars--4